extensions/store/media-source/storeDefinition.js

```diff
@@ -0,0 +1,57 @@
+const DEFAULT_STATE = {
+	players: {},
+};
+
+const defaultMediaStatus = {
+	status: 'is-paused',
+	position: 0,
+};
+
+const actions = {
+	registerMediaSource( id, mediaStatus ) {
+		return {
+			type: 'REGISTER_MEDIA_SOURCE',
+			id,
+			status: { ...defaultMediaStatus, ...mediaStatus },
+		};
+	},
+
+	unregisterMediaSource( id ) {
+		return {
+			type: 'UNREGISTER_MEDIA_SOURCE',
+			id,
+		};
+	},
+};
+
+const selectors = {};
+
+const storeDefinition = {
+	reducer( state = DEFAULT_STATE, action ) {
+		switch ( action.type ) {
+			case 'REGISTER_MEDIA_SOURCE':
+				return {
+					...state,
+					players: {
+						...state.players,
+						[ action.id ]: action.status,
+					},
+				};
+
+			case 'UNREGISTER_MEDIA_SOURCE':
+				// eslint-disable-next-line no-case-declarations
+				const currentState = Object.assign( {}, state );
+				if ( currentState.players[ action.id ] ) {
+					delete currentState.players[ action.id ];
+				}
+				return currentState;
```

Comment:
```suggestion
			case 'UNREGISTER_MEDIA_SOURCE': {
				const currentState = Object.assign( {}, state );
				if ( currentState.players[ action.id ] ) {
					delete currentState.players[ action.id ];
				}
				return currentState;
			}				
```
Adding brackets should fix the lint error and prevent future conflicts if other action types have a `currentState`.